plot_output.gnuplot

```diff
@@ -5,4 +5,5 @@ unset xtics
 unset ytics
```

Comment:
The functionality of this file is redundant with plot.sh, but this file is still in use for rendering the random path example.